Extension/src/LanguageServer/extension.ts

```diff
@@ -806,14 +806,9 @@ function onSwitchHeaderSource(): void {
                     vscode.window.showTextDocument(document, editor.viewColumn);
                 }
             });
-            // TODO: Handle non-visibleTextEditor...not sure how yet.
+
             if (!foundEditor) {
-                if (vscode.window.activeTextEditor !== undefined) {
-                    // TODO: Change to show it in a different column?
-                    vscode.window.showTextDocument(document, vscode.window.activeTextEditor.viewColumn);
```

Comment:
Wasn't this logic intended to address a previous bug here?
Does this fix also address? :  https://github.com/microsoft/vscode-cpptools/issues/2667

Reply:
Yeah -- it was intended to fix the bug that VS Code fixed via adding the revealIfOpen setting. This method leaves the behavior up to VS and the user setting instead of forcing it to use the activeTextEditor.viewColumn.

Yes, this fixes https://github.com/microsoft/vscode-cpptools/issues/2667, i.e. they're duplicate issues. I couldn't locate that.

Reply:
It looks like VS Code added revealIfOpen as a mitigation for https://github.com/microsoft/vscode/issues/15178, which is the request to add a more powerful "open editors" API.